### DISCUSSION TITLE
Could you explain what 'UNK' is?

### DISCUSSION THREAD

**Discussion by flrina0305** [seominjoon/kaist-ai605#5] [Q&A] [answered]
In problem2.1: Make sure that you add an UNK token to the vocabulary to account for words (during inference time) that you haven't seen.

Could you explain the role of 'UNK' token?

Thanks in advance.

**Comment by seominjoon** [comment, marked as answer]
Suppose your vocabulary is ["Hello", "world"] during training time. If you see a new word during inference time, e.g. "hi", then you need a default embedding for the new word. This is often times handled by "UNK", which basically maps every word that you haven't seen during training time.

**Reply by flrina0305** [reply]
Thank you!

**Reply by seominjoon** [reply]
Please mark as answered if it is :)

**Reply by flrina0305** [reply]
> Please mark as answered if it is :)

yep!
